bin/emqx

```diff
@@ -199,6 +199,11 @@ usage() {
 
 COMMAND="${1:-}"
 
+if [ -z "$COMMAND" ]; then
```

Comment:
Nit: `-z "${COMMAND+x}"` won't trip "unused variable" warning, if it's enabled.

Reply:
it's set at line 200.
this line is for regression purpose to ensure COMMAND is set before everything else.